### ISSUE TITLE
Any plan to support hot reload modified  classes/resource ?

### ISSUE THREAD

**Issue by surajchhetry** [kumuluz/kumuluzee#137]
Hi,
When we do any change on classes/resources we have to build it and re run it . Like other Micro-profile implementation ( quarkus ) are we planning to support hot reload modified classes/resource ?

**Comment by MBJuric** [comment]
Hi,
KumuluzEE is used by several companies for their production systems, where high security level is valuable. Not being able to change code in runtime is an important security aspect for most clients. 
However, we will have a look at quarkus and possible add this as an optioanl feature.

**Comment by surajchhetry** [comment]
You are right but this is only available in development mode with the help of maven plugin. In quarkus when you run maven command 
  `mvn compile quarkus:dev` 
now when you hit the REST end point , it will check if it was modified or not. If modified it will compile and reload the resource. Hence increase the developer productivity . KumuluEE being my favorite Micro-profile implementation I was just suggesting .

**Comment by surajchhetry** [comment]
> Hi,
> KumuluzEE is used by several companies for their production systems, where high security level is valuable. Not being able to change code in runtime is an important security aspect for most clients.
> However, we will have a look at quarkus and possible add this as an optioanl feature.

It is just a devtool no need to be in production .

**Comment by pierre-gautier** [comment]
Currently trying out Kumuluzee with Eclipse IDE, when running mvn kumuluzee:run from Eclipse, when the class are changed their are injected and used live
